index.html

```diff
@@ -0,0 +1,330 @@
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+              <p>
+                A daily selection of privately personalized; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button class="btn">See Project</button>
+          </div>
+        </div>
+      </section>
+      <!-- End portfolio section -->
+      <!-- Start About section -->
+      <section id="about">
+        <div class="container">
+          <div class="about-content">
+            <div class="about-main-content">
+              <h2>About Myself</h2>
+              <p>
+                I'm a software developer! I can help you build a project,
+                feature or website look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don't hesitate to contact me.
+              </p>
+            </div>
+            <!-- End hero main content section -->
+            <!-- Start hero social section -->
+            <div class="about-social">
+              <p>Let's connect</p>
+              <ul>
+                <li>
+                  <a href="#" target="_blank" rel="noopener noreferrer">
+                    <i class="fa-brands fa-twitter"></i>
+                  </a>
+                </li>
+                <li>
+                  <a href="#" target="_blank" rel="noopener noreferrer">
+                    <i class="fa-brands fa-linkedin-in"></i>
+                  </a>
+                </li>
+                <li>
+                  <a href="#" target="_blank" rel="noopener noreferrer">
+                    <i class="fa-solid fa-m"></i>
+                  </a>
+                </li>
+                <li>
+                  <a href="#" target="_blank" rel="noopener noreferrer">
+                    <i class="fa-brands fa-github"></i>
+                  </a>
+                </li>
+                <li>
+                  <a href="#" target="_blank" rel="noopener noreferrer">
+                    <i class="fa-regular fa-hand-peace"></i>
+                  </a>
+                </li>
+              </ul>
+              <button class="btn">Get my resume</button>
+            </div>
+            <div class="dropdown-wrapper">
+              <ul class="modals">
+  <li>
+    <span>Languages</span>
+    <i class="fa-solid fa-arrow-down"></i>
+    <ul>
+      <li>
+        <img src="images/javascript.png" alt="" />
+        <p>JavaScript</p>
+      </li>
+      <li>
+        <img src="images/html.png" alt="" />
+        <p>HTML</p>
+      </li>
+      <li>
+        <img src="images/css.png" alt="" />
+        <p>CSS</p>
+      </li>
+    </ul>
+  </li>
+  
+  <li>
+    <span>Frameworks</span>
+    <i class="fa-solid fa-arrow-right"></i>
+  </li>
```

Comment:
- [ ] Please try as much as possible to stick to the Figma design. This includes ensuring that the images have the same height as Figma's design. In addition, the color of the texts should match the Figma design. This to ensure the portfolio is more professional

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/228253170-21dfd7ca-3ec7-47b1-93e6-ba2704b60300.png)

Reply:
Can you kindly elaborate. Im not too quite sure yet

Reply:
Kindly try as much as possible to be specific to the question you are asking. It will help me to give further support. Kindly mention me in the thread to get an alert.